WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -221,6 +221,7 @@ public void setArguments(Bundle args) {
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        AnalyticsTracker.track(AnalyticsTracker.Stat.READER_SEARCH_LOADED);
```

Comment:
This fragment isn't used just for search results, to this shouldn't be tracked here.